### ISSUE TITLE
Disable `Allow blob public access` for load generator storage accounts

### ISSUE THREAD

**Issue by heoelri** [Azure/Mission-Critical-Online#1240]
A couple of storage accounts are getting flagged. This contains the global storage account where this is needed as it hosts images, but also the storage accounts used by the loadgenerator. Is this needed or can we disable public blob access for the loadgenerator storage accounts? 

> **In-Scope** All Storage Account types, except for File Storage.
> **Out-of-Scope** File Storage Account types. 
> **Logic** If Allow Blob public access is disabled, then the control will pass.Note: ​​​​​​​This control will only be evaluated if the provisioning state is marked as 'Succeeded'. 

@sebader thoughts?

**Comment by sebader** [comment]
For the load generators I'm pretty sure that we can disable it without any issues

**Comment by sebader** [comment]
once this is merged, please redeplo the loadgenerator and lets make sure it runs

**Comment by heoelri** [comment]
> once this is merged, please redeploy the loadgenerator and lets make sure it runs

Yes, needed to check if INT completes successfully first.
